### PR TITLE
Make doc structure consistent and up-to-date

### PR DESCRIPTION
**Pull request by Julian-O**
This is part of an effort to make the Kivy sibling projects' documentation structure consistent and up-to-date.

Unrelated items:
* Readme was vestigial. Wrote description from scratch.

CHECKLIST

* CONTRIBUTING.md
   [x] If repo takes user contributions, is present
   [x] In root dir (not .github dir)
   [x] Explains relationship to Kivy, if unclear.
   [x] Refers to kivy/kivy Contribution Guidelines.
   
* LICENSE
   [x] If repo takes user contributions, is present.
   [x] Acknowledges the range of years to 2023.
   [x] Acknowledges Kivy Team and other contributors
   [x] Mentions it is an MIT License.
   
* CODE_OF_CONDUCT.md
   [x] If repo takes user contributions or hosts discussions, is present.
   [x] Refers to kivy/kivy CODE_OF_CONDUCT.md
   
* CONTACT.md
   [x] Refers to kivy/kivy CONTACT.md

* FAQ.md
   [NA] If repo is big enough for RST documentation, is present.
   
* README:
   [x] Is a Markdown file.
   [x] Describes the project.
   [x] Describes its place in the Kivy sibling projects.
   [NA] If Documentation exists, mention it.
   [x] If CONTRIBUTING exists, mentions it.
   [x] If LICENSE exists, mentions it.
   [x] If CODE_OF_CONDUCT exists, mentions it.
   [x] Mentions kivy/kivy CONTACT.md
   [NA] Uses Python syntax colouring for embedded Python code.
   [x] Uses badges to display current status, including:
        [x] Backers
		[x] Sponsors
		[x] GitHub contributors
		[x] Contributor Covenant
		[NA] PyPI Version
		[NA] PyPI Python Version
		[x] Build/Test status

   [x] Displays all contributors to the repo.		
   [x] Displays backers
   [x] Displays top sponsors.

* RST documentation, if present
   [NA] Describes the project.
   [NA] Describes its place in the Kivy sibling projects.
   [NA] Mentions (Kivy/Kivy) Contact Us link.
   [NA] Mentions LICENSE.
   [NA] Mentions CONTRIBUTING
   [NA] Mentions FAQ
   [NA] conf.py mentioned Kivy Team and other contributors
		- copyright, latex_documents, man_pages, texinfo documents
   
   
* WORKFLOWS
   [NA] NO_RESPONSE.yml is present if the repo has awaiting_reply tag.
   [NA] NO_RESPONSE uses latest script versions.
   [NA] SUPPORT.yml is present if the repo has a `support` tag.
   [NA] SUPPORT.yml refers to repo's CONTACT.md

* setup.py/cfg, if present and on PyPI
   Note: Template used for Kivy SDK for Windows 
   [too hard] Supplies description to PyPI
   [too hard] Supplies Python versions to PyPI
   [x] Supplies Documentation, if any, to PyPI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,161 @@
...
 
-This repository contains scripts for Kivy SDK generation on Windows, OS X and Linux.
+Kivy Software Development Kit (Kivy SDK) is an installable package of all the 
+basic dependencies needed to run [Kivy framework](https://kivy.org) apps.
+
+Installing them as a single package may save time.
+
+The Kivy SDK Packager (kivy-sdk-packager) repository contains
+the definitions of the SDK for each of three different platforms, and the tools 
+to build and distribute them.
```
Comment (misl6) [inline README.md:11]: Not properly, `kivy-sdk-packager` has multiple purposes (and I wonder if would be better to split it into multiple repos in the future).

- Windows:
    - We have scripts to build dependencies to run Kivy (Such as `sdl2`, `glew`and `angle`)
 - Linux (debian):
    - `kivy-sdk-packager` hosts the files needed during the execution of the build on Ubuntu PPA
 - macOS:
     - `kivy-sdk-packager` hosts the scripts needed to package a `.dmg` or `.app` (can be the `Kivy.app` we distribute when a new Kivy release is produced, or can be used by our users to package their Kivy-based app for macOS)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,161 @@
...
+
+This repository contains scripts for Kivy SDK generation on Windows, macOS and
+Linux.
+
+Kivy SDK Packager is managed by the [Kivy Team](https://kivy.org/about.html).
+
+[![Backers on Open Collective](https://opencollective.com/kivy/backers/badge.svg)](#backers)
+[![Sponsors on Open Collective](https://opencollective.com/kivy/sponsors/badge.svg)](#sponsors)
+[![GitHub contributors](https://img.shields.io/github/contributors-anon/kivy/kivy-sdk-packager)](https://github.com/kivy/kivy-sdk-manager/graphs/contributors)
```
Comment (misl6) [inline README.md:20]:
```suggestion
[![GitHub contributors](https://img.shields.io/github/contributors-anon/kivy/kivy-sdk-packager)](https://github.com/kivy/kivy-sdk-packager/graphs/contributors)
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,161 @@
...
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
+
+## Linux (Debian)
+
+Kivy SDKs for Linux are distributed as 
+[Personal Package Archives](https://launchpad.net/ubuntu/+ppas) (PPA) files.
+They are built and hosted by [Launchpad](https://launchpad.net/) to the
+specifications (recipes) provided here.
```
Comment (misl6) [inline README.md:28]: See above (the explanation about `kivy-sdk-packager`)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,161 @@
...
+| Version      | SDK Binary                                                                                           | Source                                                                                          |
+|--------------|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| Stable Build | [Launchpad PPA packages](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy/+packages)       | [Launchpad recipes](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy/+packages)       |
+| Daily Build  | [Launchpad PPA packages](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy-daily/+packages) | [Launchpad recipes](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy-daily/+packages) |
+
+[Kivy SDK for Linux README](linux/debian/README.md)
+
+## macOS
+
+Kivy SDKs for macOS are built as Disk Image (DMG) files.
```
Comment (misl6) [inline README.md:39]: See above (the explanation about `kivy-sdk-packager`)

File: README.md [hunk 5]
```diff
@@ -1,4 +1,161 @@
...
+[Buildozer](https://buildozer.readthedocs.io) builds and uses Kivy SDKs to 
+package macOS apps.
+
+[![macOS Tests](https://github.com/kivy/kivy-sdk-packager/actions/workflows/test_macos.yaml/badge.svg)](https://github.com/kivy/kivy-sdk-packager/actions/workflows/test_macos.yaml)
+
+[Kivy SDK for macOS README](osx/README.md)
+
+## Windows
+
+Kivy SDK for Windows are built as wheels that can be installed via
```
Comment (misl6) [inline README.md:57]: Is not an SDK (see explanation above)

File: README.md [hunk 6]
```diff
@@ -1,4 +1,161 @@
...
+
+Four variants are released, based on four different 
+[OpenGL ES](https://en.wikipedia.org/wiki/OpenGL_ES) implementations:
+
+| Version                                                                 | PyPI Name                                                            |
+|-------------------------------------------------------------------------|----------------------------------------------------------------------|
+| [Angle](https://chromium.googlesource.com/angle/angle/+/main/README.md) | [kivy-deps.angle](https://pypi.org/project/kivy-deps.angle/)         |
+| [Glew](https://glew.sourceforge.net/)                                   | [kivy-deps.glew](https://pypi.org/project/kivy-deps.glew/)           |
+| [Gstreamer](https://gstreamer.freedesktop.org/)                         | [kivy-deps.gstreamer](https://pypi.org/project/kivy-deps.gstreamer/) |
+| [SDL2](https://www.libsdl.org/)                                         | [kivy-deps.sdl2](https://pypi.org/project/kivy-deps.sdl2/)           |
```
Comment (misl6) [inline README.md:69]: This is not true 😅

See:
- https://kivy.org/doc/stable/api-kivy.graphics.cgl.html
- https://kivy.org/doc/stable/gettingstarted/installation.html#installing-kivy-s-dependencies

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi @Julian-O .

I left some comments, and we need to fix the README before the merge.
If you need some more explanations, feel free to ping me on Discord.

**Important note:**
Except for macOS scripts, we do not expect "standard users" to use any other feature of `kivy-sdk-packager` directly.

**Review by misl6 — CHANGES_REQUESTED** [review]
We're almost there 🏁

I've left some minor comments regarding typos, and outdated content.

**Comment by Julian-O** [comment]
@misl6: Thanks! Done.

Great news about setting the deploy target. Removed warning from the macOS README too.

**Pull request merged**